src/dso_api/dynamic_api/serializers.py

```diff
@@ -286,6 +256,23 @@ def _raise_better_embed_error(cls, field_name, prefix=""):
                     f" for embedding as it's a summary of a huge listing."
                 ) from None
 
+    def get_embedded_objects_by_id(
+        self, embedded_field: AbstractEmbeddedField, id_list: list[Union[str, int]]
+    ) -> Union[models.QuerySet, Iterable[models.Model]]:
+        """Retrieve a number of embedded objects by their identifier.
+
+        This override makes sure the correct temporal slice is returned.
+        """
+        if embedded_field.is_loose:
+            # Special handling for embedding temporal relationships, if they are loose relations
+            model = embedded_field.related_model
+            id_field = embedded_field.related_id_field or model.table_schema().identifier[0]
```

Comment:
Re: `model.table_schema().identifier[0]` Why are you only interested in the the first element of `identifier`and not the others?

Reply:
This is what the old code also does. If there is a compound key `["identifier", "volgnummer"]`, only the grouping first key will be used here.

Reply:
I've added a comment:

`# When 'identifier' is ["identificatie", "volgnummer"], take the first as grouper.`